maven-plugins/run-automation-package-sample/pom.xml

```diff
@@ -0,0 +1,223 @@
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <!-- Configuration for OS version (maven profile) -->
+    <profiles>
+        <profile>
+            <id>StepOS</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>ch.exense.step</groupId>
+                            <artifactId>step-os-maven-plugin</artifactId>
+                            <version>${step-maven-plugin.version}</version>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+
+                <plugins>
+                    <plugin>
+                        <groupId>ch.exense.step</groupId>
+                        <artifactId>step-os-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>deploy-packaged-to-step</id>
+                                <phase>package</phase>
+                                <configuration>
+                                    <url>${step-server-url}</url>
+
+                                    <artifactClassifier>tests</artifactClassifier>
+                                    <description>My test execution</description>
+
+                                    <waitForExecution>true</waitForExecution>
+                                    <ensureExecutionSuccess>true</ensureExecutionSuccess>
+
+                                    <!-- Custom execution parameters -->
+                                    <executionParameters>
+                                        <env>PROD</env>
+                                        <PARAM_EXEC>Value</PARAM_EXEC>
+                                        <PARAM_EXEC2>Value</PARAM_EXEC2>
+                                        <PARAM_EXEC3>Value</PARAM_EXEC3>
+                                    </executionParameters>
+                                </configuration>
+                                <goals>
+                                    <goal>run-packaged-automation-packages-os</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <!-- Configuration for EE version (maven profile) -->
+        <profile>
+            <id>StepEE</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>ch.exense.step</groupId>
+                            <artifactId>step-ee-maven-plugin</artifactId>
```

Comment:
This will not be pushed to Sonatype and thus requires our nexus-exense repository

    <pluginRepository>
        <id>nexus-exense</id>
        <url>https://nexus-enterprise.exense.ch/repository/exense/</url>
    </pluginRepository>

Reply:
Both OS and EE versions of maven plugins are located in `Step OS` repository. Do we need some `pluginRepository` configuration in this case?

Reply:
@jeromecomte I let you check and mark this as resolved, but indeed both plugins are in OS code base and will be available in central (OS) repository